x-pack/plugins/cases/common/api/runtime_types.ts

```diff
@@ -13,6 +14,9 @@ import { isObject } from 'lodash/fp';
 
 type ErrorFactory = (message: string) => Error;
 
+export const OmitProp = <O extends rt.Props, K extends keyof O>(o: O, k: K): Omit<O, K> =>
```

Comment:
nit -> to avoid the used of omit, we can always define the type without it first and just extends from this one to add a new attributes.

Reply:
Added to the issue here: https://github.com/elastic/kibana/issues/100804